src/graphql/search/filters/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Filter, FilterOptions } from '../../../generated/graphql'
+
+// Combine filter options and return the new filteroptions
+export function combineFilterOptions(options: Array<FilterOptions>): Array<FilterOptions> {
+  return options.reduce((acc: any, cur: any) => {
```

Comment:
The parameters here are still of type `any`. It's possible to pass the type to the initial value so the type here can be removed:

```ts
// No need for a return type, can be inferred
export function combineFilterOptions(options: Array<FilterOptions>) {
  return options.reduce((acc, cur) => {
    // TypeScript knows that `acc` and `cur` are of type `FilterOption` because the intial value has a type.
  }, [] as Array<FilterOptions>)
}
```